### PR TITLE
Add upsell placeholder for Download as PDF

### PR DESCRIPTION
**Pull request by truongwp**
Fixes https://github.com/Strategy11/formidable-pro/issues/4358

### CODE REVIEW

File: classes/helpers/FrmEntriesHelper.php [hunk 1]
```diff
@@ -708,6 +718,29 @@ private static function get_action_links( $id, $entry ) {
...
+	/**
+	 * Gets data attributes for PDFs addon upgrade link.
+	 *
+	 * @return array
+	 */
+	private static function get_pdfs_upgrade_link_data() {
+		$data = array(
+			'oneclick' => '',
+			'requires' => '',
+			'upgrade'  => __( 'Forms to PDF', 'formidable-pro' ),
```
Comment (truongwp) [inline classes/helpers/FrmEntriesHelper.php:730]: @Crabcyborg Maybe you wil want to update this and `medium`.

**Comment by Crabcyborg** [comment]
Thanks for working on this @truongwp!

It looks like the save and reload text gets added to the modal when you active the PDFs add on, but the buttons aren't there. Could we get the buttons working, or get that text about saving and reloading to go away?

Screen Shot 2023-08-09 at 3 55 28 PM

**Comment by truongwp** [comment]
@Crabcyborg Thank you. I decided to remove that part. Look like Save and Reload just works on the Settings and Form builder page.

**Review by Crabcyborg — APPROVED** [review]
Thanks @truongwp! I made a small commit to give the medium attribute a more detailed description so we can distinguish it from another pdfs upsell.

https://github.com/Strategy11/formidable-forms/pull/1259/commits/2b5acc8b152d24416d890fbd89258bddeb97e057

**Pull request merged**